docs/source/guides/variables.rst

```diff
@@ -35,7 +41,7 @@ This is done by calling its :meth:`~Variable.backward` method:
    >>> y.backward()
 
 This runs *error backpropagation* (a.k.a. *backprop* or *reverse-mode automatic differentiation*).
-Then, the gradient is computed and stored in the :attr:`~Variable.grad` attribute of the input variable ``x``:
+Then, the gradient is computed and stored in the :attr:`~chainer.Variable.grad` attribute of the input variable ``x``:
```

Comment:
Should we mention `grad_var`?

Reply:
Thanks, that makes sense. I added `Higher-Order Derivatives` section to show the usage of `grad_var`.

Reply:
Great! This was never documented before I guess.